src/app/admin/users/PageUserUpdate.tsx

```diff
@@ -37,28 +39,39 @@ export const PageUser = () => {
     isLoading: userIsLoading,
     isFetching: userIsFetching,
   } = useUser(login, { refetchOnWindowFocus: false });
-  const queryClient = useQueryClient();
+
+  const form = useForm({ subscribe: false });
 
   const toastSuccess = useToastSuccess();
   const toastError = useToastError();
 
   const { mutate: editUser, isLoading: editUserIsLoading } = useUserUpdate({
-    onError: (error: any) => {
-      const { title } = error?.response?.data || {};
+    onError: (error: AxiosError) => {
+      const { title, errorKey } = error.response.data;
       toastError({
-        title: 'Update failed',
+        title: t('users:update.feedbacks.updateError.title'),
         description: title,
       });
+      switch (errorKey) {
+        case 'userexists':
+          form.invalidateFields({
+            login: t('users:data.login.alreadyUsed'),
+          });
+          break;
+        case 'emailexists':
```

Comment:
same

Reply:
Yes, can you open an issue because is not related to translations? :)